src/date-picker/date-picker.jsx

```diff
@@ -149,11 +159,11 @@ const DatePicker = React.createClass({
 
   getDefaultProps() {
     return {
-      formatDate: DateTime.format,
+      formatDate: toIsoDateString,
```

Comment:
Why not changing this https://github.com/callemall/material-ui/blob/master/src/utils/date-time.js#L137?